administrator/components/com_modules/Model/SelectModel.php

```diff
@@ -97,14 +97,15 @@ protected function getListQuery()
 		$query->from($db->quoteName('#__extensions') . ' AS a');
 
 		// Filter by module
-		$query->where('a.type = ' . $db->quote('module'));
+		$query->where($db->quoteName('a.type') . ' = ' . $db->quote('module'));
 
 		// Filter by client.
-		$clientId = $this->getState('client_id');
-		$query->where('a.client_id = ' . (int) $clientId);
+		$clientId = (int) $this->getState('client_id');
+		$query->where($db->quoteName('a.client_id') . ' = :clientid')
+			->bind(':clientid', $clientId);
```

Comment:
```suggestion
			->bind(':clientid', $clientId, ParameterType::INTEGER);
```
Add `use Joomla\Database\ParameterType;`